cmd/thriftbreak/main_test.go

```diff
@@ -32,58 +32,86 @@ import (
 )
 
 func TestThriftBreakIntegration(t *testing.T) {
-	tmpDir := t.TempDir()
-	from := map[string]string{
-		"v1.thrift": "namespace rb v1\n" +
-			"struct AddedRequiredField {\n" +
-			"    1: optional string A\n" +
-			"    2: optional string B\n" +
-			"}\n" +
-			"\nservice Foo {\n    void methodA()\n}",
-		"test/v2.thrift": `service Bar {}`,
-		"test/c.thrift":  `service Baz {}`,
-		"test/d.thrift": `include "../v1.thrift"
-		service Qux {}`, // d.thrift will be deleted below.
-		"somefile.go": `service Quux{}`, // a .go file, not a .thrift.
+	type test struct {
+		desc      string
+		want      string
+		gitDirArg string
+		extraCmd  string
 	}
-	// For c.thrift we are also checking to make sure includes work as expected.
-	to := map[string]string{
-		"v1.thrift": "namespace rb v1\n" +
-			"struct AddedRequiredField {\n" +
-			"    1: optional string A\n" +
-			"    2: optional string B\n" +
-			"    3: required string C\n}\n" +
-			"service Foo {}",
-		"test/v2.thrift": `service Foo {}`,
-		"test/c.thrift": `include "../v1.thrift"
-		service Bar {}`,
-		"somefile.go": `service Qux{}`,
+	tests := []test{
```

Comment:
Naming this type is unnecessary. For table tests, we typically don't name the test case struct unless we need to declare a separate `func` that accepts it.

```
tests := []struct {
  // fields
}{
  // test cases
}
```